### PR TITLE
More automobiles (reupload)

### PR DESCRIPTION
**Pull request by Lengord**
## About The Pull Request
I still had sprites from Cakey request for vehicles sprites, so why not to try. Copy-pasted some code, seems work ok'ish, you can drive them atleast. Also added cramped interriors for them. (i broke it, so reupload)

## Why It's Good For The Game
If it's bad for game atleast can be used for existing props, or for events. 

## Changelog

:cl:
add: Added more vehicles to drive, and interrior maps
imageadd: Added direction sprites for some vehicles
code: Added code to make new vehicles and hardpoints plus spawners for them
/:cl:

### CODE REVIEW

**Comment by harryob** [comment]
make sure your changelog is descriptive as to what you're actually changing - not just changed some code

**Review by Nanu308 — COMMENTED** [review]
mapping lgtm

File: code/modules/vehicles/hardpoints/wheels/locomotion.dm [hunk 1]
```diff
@@ -61,3 +61,28 @@
...
+				/obj/item/hardpoint/locomotion/truck/wheels/civvan)
+
+/obj/effect/hardpoint_spawner/locomotion/Initialize()
+	. = ..()
+	spawn_hardpoint()
+	qdel(src)
+
+/obj/effect/hardpoint_spawner/locomotion/spawn_hardpoint()
+	var/I = choose_hardpoint()
+	return (new I(loc))
```
Comment (stanalbatross) [inline code/modules/vehicles/hardpoints/wheels/locomotion.dm:88]: would be easier and better OOP wise to make this a subtype of spawner/random

File: code/modules/vehicles/hardpoints/wheels/truck.dm [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/obj/item/hardpoint/locomotion/truck
+	icon = 'icons/obj/vehicles/hardpoints/truck.dmi'
+	slot = HDPT_WHEELS
+
+
+	//TRUCK Wheels
+/obj/item/hardpoint/locomotion/truck/wheels
+	name = "Military wheels"
```
Comment (stanalbatross) [inline code/modules/vehicles/hardpoints/wheels/truck.dm:8]:
```suggestion
	name = "military wheels"
```
names need to be lower case

File: code/modules/vehicles/hardpoints/wheels/truck.dm [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	health = 500
+
+	//with this settings, takes 3 tiles to reach top speed
+	move_delay = 3.8
+	move_max_momentum = 3
+	move_momentum_build_factor = 1.8
+	move_turn_momentum_loss_factor = 0.6
+
+/obj/item/hardpoint/locomotion/truck/treads/crane
+	name = "Crane treads"
```
Comment (stanalbatross) [inline code/modules/vehicles/hardpoints/wheels/truck.dm:50]: same with all these names

File: code/modules/vehicles/truck/civtruck.dm [hunk 4]
```diff
@@ -0,0 +1,104 @@
+/obj/vehicle/multitile/civtruck
+	name = "Civilian truck"
```
Comment (stanalbatross) [inline code/modules/vehicles/truck/civtruck.dm:2]: same

File: code/modules/vehicles/truck/civvan.dm [hunk 5]
```diff
@@ -0,0 +1,88 @@
+/obj/vehicle/multitile/civvan
+	name = "Civilian van"
```
Comment (stanalbatross) [inline code/modules/vehicles/truck/civvan.dm:2]: same and for all other added names

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]
code good

**Review by nauticall — CHANGES_REQUESTED** [review]
image
The only thing preventing me from giving my stamp is this van -- it's too bright red. Make it a darker red, get back to me, and we can get this baby finally merged aight?

**Pull request closed without merging**